Makefile

```diff
@@ -293,6 +295,18 @@ kind2-up: $(KIND) $(KUBECTL) $(HELM)
 kind2-down: $(KIND)
 	./hack/kind-down.sh --cluster-name gardener-local2 --path-kubeconfig $(REPO_ROOT)/example/provider-local/seed-kind2/base/kubeconfig --keep-backupbuckets-dir
 
+kind-extensions-up: $(KIND) $(KUBECTL)
```

Comment:
I created separate make targets because i would like to avoid that you could destroy your cluster accidentally with `make kind-down` when you go to lunch. In the provider-local setup this might not be a big issue, but in the extension case there would remain orphaned items in the infrastructure accounts which is a bit cumbersome to clean up.

Reply:
I got the motivation but still I think it would be nice to merge both. Maybe it wouldn't be an issue too if `make kind-down` just stopped the container and `make kind-clean` destroys it.